cmd/vcert/flags.go

```diff
@@ -423,6 +423,15 @@ var (
 		Destination: &flags.omitSans,
 	}
 
+	flagCSRFormat = &cli.StringFlag{
+		Name: "format",
+		Usage: "Generates the Certificate Signing Request in the specified format. Options include: pem | json\n" +
+			"\t\tpem: Generates the CSR in classic pem format to be used as a file.\n" +
```

Comment:
Should be capitalized PEM in the description (i.e. "classic PEM format")

Reply:
modified PEM text to be capitalized